src/nvim/ex_cmds.c

```diff
@@ -4960,3 +4961,56 @@ void ex_oldfiles(exarg_T *eap)
     }
   }
 }
+
+void ex_trust(exarg_T *eap)
+{
+  char *p = skiptowhite(eap->arg);
+  size_t len = (size_t)(p - eap->arg);
+  char *arg1 = xmemdupz(eap->arg, len);
+
+  bool success = false;
+  bool action_allow = false, action_deny = false, action_remove = false;
```

Comment:
Don't initialize multiple variables on the same line.

```suggestion
  bool action_allow = false;
  bool action_deny = false;
  bool action_remove = false;
```

Reply:
Removed in 7a93893914063e690a008b91552fbdb127c55876